packages/trader/src/sass/app/_common/form/duration.scss

```diff
@@ -0,0 +1,11 @@
+/** @define duration */
+.dc-dropdown {
+    &__display {
+        &--no-symbol-duration {
```

Comment:
@maryiafrantsava-deriv What do you think about fitting it into a single line 😊 ?
`.dc-dropdown__display--no-symbol-duration {`
*imho, in case later someone needs to add more styles to its parts, that person will split it

Reply:
@maryia-deriv, for my taste, the separated record is more readable, the spelling does not affect anything anyway. Thank you 🙂